scripts/histmakers/mz_dilepton.py

```diff
@@ -54,7 +52,7 @@
 mass_max = 120
 
 ewMassBins = theory_tools.make_ew_binning(mass = 91.1535, width = 2.4932, initialStep=0.010)
-dilepton_ptV_binning = [0, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 13, 15, 17, 20, 23, 27, 32, 40, 54, 100] if not args.finePtBinning else range(60)
+dilepton_ptV_binning = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 13, 15, 17, 20, 23, 27, 32, 40, 54, 100] if not args.finePtBinning else range(60)
```

Comment:
Is it important to add the additional bin? In the past we had the first bin with width of 2 because it has less events and the theory uncertainty is large.

Reply:
I added it to try to understand the difference between the reco-level corr and the unfolding-based ptll corr. In the end it didn't make a lot of difference. It can be removed (we should also maybe try again on making this more configurable)